features/FxA-28-change-email/README.md

```diff
@@ -0,0 +1,53 @@
+
+Change Email
+======================
+
+https://mozilla.aha.io/features/FXA-28
+
+As an FxA user when I lose control of the email account  that is my username in Firefox Accounts I want to be able to change that FxA email address so that I don't lose access to my data and other FxA-relying services.
+* When a user changes their email in Firefox Accounts, it simply updates in other clients and relying services like display name without requiring them to reconnect.
+
+## Settings Collapsed
+* When the user visits their account settings, an **Account email** section appears under the **Display name** section.
```

Comment:
We had previously talked about including this in the "communication preferences" section, but I notice that section is entirely missing from the UI mocks here (I assume due to language preference).  Doing it in a top-level section does seem better with the new design, but I just wanted to sanity-check the interaction here because we talked about it previously.